src/openms/include/OpenMS/FORMAT/Vec.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+#include <simde/x86/sse2.h>  // or the lowest needed header
+
+class Vec128 {
```

Comment:
please add proper doxygen documentation on what this class does.
Also naming it `Vec128i` would be more appropriate to allow future specification of `__m128` and `__m128d` if need be.